intro-to-R-tidyverse/01-intro_to_base_R.Rmd

```diff
@@ -386,8 +386,12 @@ question_values %in% values_1_to_20
 _Data frames are the most fundamental unit of data analysis in R._ 
 They are tables which consist of rows and columns, much like a _spreadsheet_. 
 Each column is a variable which behaves as a _vector_, and each row is an observation. 
-We will begin our exploration with the old trusted dataset `iris`, which comes with R. 
-Learn about this dataset using the standard help approach of `?iris`.
+We will begin our exploration with dataset about penguins from the `palmerpenguins` package. 
+To use this dataset, we will need to attach the package with `library()` function (more on this later).
+
+```{r penguin-library}
+library(palmerpenguins)
```

Comment:
The trouble with doing it this way is that the data frame doesn't appear in the environment pane. I think we should probably do this with an assignment and brief explanation that we are pulling the data from a package.
```suggestion
penguins <- palmerpenguins::penguins
```